ibm/mas_devops/roles/cert_manual_order/meta/main.yml

```diff
@@ -0,0 +1,22 @@
+galaxy_info:
```

Comment:
This role may fit better in ansible-fvt as it is very specific for an initial load of CNAMES in CIS to be used by Secrets Manager to generate the certificate